packages/onboarding/src/getAssertData.ts

```diff
@@ -0,0 +1,519 @@
+import Balance, { CurrencyType, DataCredits } from '@helium/currency'
+import OnboardingClient from './OnboardingClient'
+import BN from 'bn.js'
+import {
+  DC_MINT,
+  IOT_MINT,
+  MOBILE_MINT,
+  toBN,
+  HNT_MINT,
+  heliumAddressToSolPublicKey,
+} from '@helium/spl-utils'
+import { subDaoKey } from '@helium/helium-sub-daos-sdk'
+import { Connection, PublicKey, AccountInfo, LAMPORTS_PER_SOL, Cluster } from '@solana/web3.js'
+import { init as initDc } from '@helium/data-credits-sdk'
+import axios from 'axios'
+import { getBalance } from '@helium/currency-utils'
+import {
+  AccountLayout,
+  getAssociatedTokenAddress,
+  createAssociatedTokenAccountIdempotentInstruction,
+  getAccount,
+  getAssociatedTokenAddressSync,
+} from '@solana/spl-token'
+import {
+  iotInfoKey,
+  mobileInfoKey,
+  rewardableEntityConfigKey,
+} from '@helium/helium-entity-manager-sdk'
+import { init as initHem } from '@helium/helium-entity-manager-sdk'
+import * as Currency from '@helium/currency-utils'
+import { HotspotType } from './types'
+
+export const TXN_FEE_IN_LAMPORTS = 5000
+export const TXN_FEE_IN_SOL = TXN_FEE_IN_LAMPORTS / LAMPORTS_PER_SOL
+export const FULL_LOCATION_STAKING_FEE = 1000000 // $10 - does this need to be updated to $5? It's used as a fallback when something fails
```

Comment:
This can be fetched from the value on chain. You need to fetch the `RewardableEntityConfig`

Reply:
yeah, I have that happening already. This is just a fallback in case the lookup fails.